src/main/java/frc/robot/subsystems/shooterSubsystem/ShooterSubsystem.java

```diff
@@ -74,10 +74,10 @@ public ShooterSubsystem() {
 
     shooterLeftMotor.setInverted(true);
 
-    shooterLeftMotor.getPIDController().setP(Constants.ShooterConstants.SHOOTER_LEFT_P);
-    shooterLeftMotor.getPIDController().setI(Constants.ShooterConstants.SHOOTER_LEFT_I);
-    shooterLeftMotor.getPIDController().setD(Constants.ShooterConstants.SHOOTER_LEFT_D);
-    shooterLeftMotor.getPIDController().setFF(Constants.ShooterConstants.SHOOTER_LEFT_FF);
+    shooterLeftMotor.getPIDController().setP(Constants.ShooterConstants.SHOOTER_RIGHT_P);
+    shooterLeftMotor.getPIDController().setI(Constants.ShooterConstants.SHOOTER_RIGHT_I);
+    shooterLeftMotor.getPIDController().setD(Constants.ShooterConstants.SHOOTER_RIGHT_D);
+    shooterLeftMotor.getPIDController().setFF(Constants.ShooterConstants.SHOOTER_RIGHT_FF);
```

Comment:
Right left swapped?

Reply:
Made both sides use the same PID

Reply:
Add code comments to explain that this is temporary or consolidate the constants into one set